src/components/Dialog/Dialog.tsx

```diff
@@ -65,7 +65,6 @@ export class Dialog extends React.Component<IDialogProps, any> {
             ariaDescribedBy={ subText ? id + '-subText' : '' }
             onDismiss={ onDismiss }
             >
-            <div className={ dialogClassName }>
```

Comment:
Where will you apply the dialogClassName? We need this to be able to override styles.